Plots/XY/NCL_xy_12.py

```diff
@@ -0,0 +1,109 @@
+"""
+NCL_xy_12.py
+===============
+This script illustrates the following concepts:
+   - Emphasizing part of a curve in an XY plot
+   - Drawing longitude labels on the X axis
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/xy_12.ncl
+    - Original NCL plots: https://www.ncl.ucar.edu/Applications/Images/xy_12_1_lg.png and https://www.ncl.ucar.edu/Applications/Images/xy_12_2_lg.png
+"""
+
+##############################################################################
+# Import packages:
+import numpy as np
+import xarray as xr
+import matplotlib.pyplot as plt
+
+import geocat.datafiles as gdf
+from geocat.viz import util as gvutil
+
+##############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and load the data into
+# xarrays
+ds = xr.open_dataset(gdf.get("netcdf_files/uv300.nc"))
+
+##############################################################################
+# Method 1: Splitting the line into parts and coloring them differently
+U = ds.isel(time=0, lon=5, drop=True).U
+
+plt.figure(figsize=(7, 7))
+ax = plt.axes()
+
+bins = [0, 5, 20]
+# Slicing data is exclusive for the last value, to work around this we
```

Comment:
This comment is a little confusing, I would clarify what you're working around.

Reply:
Is it less confusing now that I have updated it?